content/en/docs/reference/command-line-tools-reference/kubelet-config.v1beta1.md

```diff
@@ -0,0 +1,1552 @@
+
+
+<table class="table table-striped">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+    
+
+  
+<tr><td style="white-space: pre"><code>enabled</code></br>
+<code>bool</code>
+</td>
+<td><p>enabled allows bearer token authentication backed by the tokenreviews.authentication.k8s.io API</p>
+</td>
+</tr>
+    
+  
+<tr><td style="white-space: pre"><code>cacheTTL</code></br>
+<a href="https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
+</td>
+<td><p>cacheTTL enables caching of authentication results</p>
+</td>
+</tr>
+    
+  
+</tbody>
+</table>
+    
+
+
+### `KubeletWebhookAuthorization`     {#kubelet-config-k8s-io-v1beta1-KubeletWebhookAuthorization}
+    
+
+
+
+**Appears in:**
+
+- [KubeletAuthorization](#kubelet-config-k8s-io-v1beta1-KubeletAuthorization)
+
+
+<table class="table table-striped">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+    
+
+  
+<tr><td style="white-space: pre"><code>cacheAuthorizedTTL</code></br>
+<a href="https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
+</td>
+<td><p>cacheAuthorizedTTL is the duration to cache 'authorized' responses from the webhook authorizer.</p>
+</td>
+</tr>
+    
+  
+<tr><td style="white-space: pre"><code>cacheUnauthorizedTTL</code></br>
+<a href="https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
+</td>
+<td><p>cacheUnauthorizedTTL is the duration to cache 'unauthorized' responses from the webhook authorizer.</p>
+</td>
+</tr>
+    
+  
+</tbody>
+</table>
+    
+
+
+### `KubeletX509Authentication`     {#kubelet-config-k8s-io-v1beta1-KubeletX509Authentication}
+    
+
+
+
+**Appears in:**
+
+- [KubeletAuthentication](#kubelet-config-k8s-io-v1beta1-KubeletAuthentication)
+
+
+<table class="table table-striped">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+    
+
+  
+<tr><td style="white-space: pre"><code>clientCAFile</code></br>
+<code>string</code>
+</td>
+<td><p>clientCAFile is the path to a PEM-encoded certificate bundle. If set, any request presenting a client certificate
+signed by one of the authorities in the bundle is authenticated with a username corresponding to the CommonName,
+and groups corresponding to the Organization in the client certificate.</p>
+</td>
+</tr>
+    
+  
+</tbody>
+</table>
+    
+
+
+### `ResourceChangeDetectionStrategy`     {#kubelet-config-k8s-io-v1beta1-ResourceChangeDetectionStrategy}
+    
+(Alias of `string`)
+
+
+**Appears in:**
+
+- [KubeletConfiguration](#kubelet-config-k8s-io-v1beta1-KubeletConfiguration)
+<p>ResourceChangeDetectionStrategy denotes a mode in which internal
+managers (secret, configmap) are discovering object changes.</p>
+
+
+
+    
+  
+  
+    
+
+### `LoggingConfiguration`     {#LoggingConfiguration}
```

Comment:
I'd expect this would be more like:
```markdown
### `LoggingConfiguration`     {#kubelet-config-k8s-io-LoggingConfiguration}
```

Reply:
This anchor is local to the page, not a global identifier for this type. It is hard to parse types referenced from existing types: some are embedded, some are referenced, some are borrowed (from a different group version).